test/unit/controllers/actions/JourneyRefinerSpec.scala

```diff
@@ -52,14 +53,14 @@ class JourneyRefinerSpec extends WordSpec with MustMatchers with MockitoSugar wi
   }
 
   "refine" should {
-    "permit request" when {
-      "answers found" in {
+    "add cache to the request" when {
+      "cache and answers are both not empty" in {
```

Comment:
I'd re-phrase this line. If answers are non-empty, then cache per-se is non-empty as well.
Maybe something like "cache contains non-empty answers" ?